build/tasks/validate.js

```diff
@@ -84,16 +105,20 @@ function createSchemas() {
 					messages: {
 						required: true,
 						type: 'object',
-						properties: {
-							fail: {
-								required: true,
-								type: 'string'
-							},
-							pass: {
-								required: true,
-								type: 'string'
-							}
+						conform: hasTwoOutcomes,
+						messages: {
+							conform: 'Must have at least two valid messages'
 						}
+						// propferties: {
```

Comment:
minor: 🤣 `propferties`, know it is commented.... delete instead?

Reply:
yeah, delete this comment